src/styles/themes/default.scss

```diff
@@ -1,110 +1,110 @@
-@import "~@/styles/partials/variables";
+@import "~styles/partials/variables";
 
 @font-face {
```

Comment:
remove fonts from 'default' file to 'fonts'-like and collect all font-relative declarations there